src/components/TodoItem.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useContext, useState } from 'react';
+import classNames from 'classnames';
+import { OrderItems } from '../types/Type';
+import { TodoContext } from '../context/TodosContext';
+
+interface Props {
+  todo: OrderItems;
+}
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const { orderItems, setOrderItems } = useContext(TodoContext);
```

Comment:
Create a hook s you don't need to pass context every time
`const useTodos = () => useContext(TodoContext);`